pythoncz_tests/views_test.py

```diff
@@ -0,0 +1,202 @@
+from urllib.parse import quote_plus as url_quote_plus
+
+import pytest
+from flask import url_for
+from werkzeug.contrib.cache import NullCache
+
+from pythoncz import app
+from pythoncz.models import github as github_module
+
+
+def create_issue(**kwargs):
```

Comment:
This sounds like the code actually creates the issue on GitHub. `fake_issue()` maybe?